diff --git a/x.c b/x.c
index d19bb3f..c9f2057 100644
--- a/x.c
+++ b/x.c
@@ -1618,7 +1618,7 @@ xdrawcursor(int cx, int cy, Glyph g, int ox, int oy, Glyph og)
 			break;
 		}
 	} else {
-		XftDrawRect(xw.draw, &drawcol,
+/*		XftDrawRect(xw.draw, &drawcol,
 				win.hborderpx + cx * win.cw,
 				win.vborderpx + cy * win.ch,
 				win.cw - 1, 1);
@@ -1634,6 +1634,7 @@ xdrawcursor(int cx, int cy, Glyph g, int ox, int oy, Glyph og)
 				win.hborderpx + cx * win.cw,
 				win.vborderpx + (cy + 1) * win.ch - 1,
 				win.cw, 1);
+*/
 	}
 }
 
